service/src/main/java/uk/gov/hmcts/reform/fpl/utils/ElementUtils.java

```diff
@@ -27,7 +27,7 @@ private ElementUtils() {
         return Stream.of(elements)
             .filter(Objects::nonNull)
             .map(element -> Element.<T>builder().value(element).build())
-            .collect(toUnmodifiableList());
+            .collect(toList());
```

Comment:
Reason for the change? Seems unnecessary

Reply:
Iit's needed for updating document history